a/html/oki/panel/index.html

```diff
@@ -0,0 +1,157 @@
+          <p><strong>Note:</strong> this will cause the ribbon to appear in front of whatever is on the page. To avoid this, see below method.</p>
+          <h3>Relative Position</h3>
+          <p>This will add the ribbon into the flow of the document. First add the following immediately after the opening <code>&lt;body&gt;</code> tag:</p>
+          <pre><code class="language-markup">&lt;div id="ok-panel" class="closed"&gt;&lt;div class="container"&gt;&lt;iframe src="//a.okfn.org/html/oki/panel/panel.html" scrolling="no"&gt;&lt;/iframe&gt;&lt;/div&gt;&lt;/div&gt;</code></pre>
+          <p>Then add this where you want the ribbon to appear (should be near the top of the page):</p>
+          <pre><code class="language-markup">&lt;a class="ok-ribbon" href="//okfn.org/"&gt;&lt;img src="//a.okfn.org/html/oki/panel/assets/images/oki-ribbon.png" alt="Open Knowledge"&gt;&lt;/a&gt;</code></pre>
+          <hr>
+          <h2>Colour Options</h2>
+          <p>With the above, default implementation, the ribbon will be green. There are also black and white options. For white on black add <code>black</code> to the <code>ok-ribbon</code> class, like so:</p>
+          <pre><code class="language-markup">&lt;a class="black ok-ribbon" href="//okfn.org/"&gt;&lt;img src="//a.okfn.org/html/oki/panel/assets/images/oki-ribbon.png" alt="Open Knowledge"&gt;&lt;/a&gt;</code></pre>
+          <p>and for black on white, add <code>white</code>:</p>
+          <pre><code class="language-markup">&lt;a class="white ok-ribbon" href="//okfn.org/"&gt;&lt;img src="//a.okfn.org/html/oki/panel/assets/images/oki-ribbon.png" alt="Open Knowledge"&gt;&lt;/a&gt;</code></pre>
+        </article>
+        <aside class="col-md-3">
+          <div class="panel panel-info">
+            <div class="panel-heading">
+              <h3 class="panel-title">Want the Code?</h3>
+            </div>
+            <div class="panel-body">
+              <a href="https://github.com/okfn/a/tree/master/a/html/oki/panel/">View on Github</a>
+            </div>
+          </div>
+          <div class="panel panel-info">
+            <div class="panel-heading">
+              <h3 class="panel-title">Using Wordpress?</h3>
+            </div>
+            <div class="panel-body">
+              <a href="https://github.com/okfn/ok-panel/tree/wordpress">Get plugin here</a>
+            </div>
+          </div>
+        </aside>
+      </div>
+    </div>
+
+    <footer class="site-footer">
+      <div class="container">
+        <a class="footer-logo" href="https://okfn.org/">
+          <img src="https://bytebucket.org/okfn/assets.okfn.org/raw/88b24904b8ecded5e6d530739743162d1c5b3d93/p/okfn/img/okfn-logo-landscape-white.png" alt="Open Knowledge">
```

Comment:
Ah yes, sorry (I was confused by the Github interface for a moment there). I'll correct that.